assets/src/styles/blocks.scss

```diff
@@ -32,3 +32,18 @@
 // Other
 @import "blocks/WideBlocks";
 @import "blocks/Editor";
+
+.is-style-small-padding {
+  padding: 8px;
```

Comment:
Can you please pay attention to the essence of the PR while it's still in draft?

Reply:
I chose these values because I quickly wanted to write an example, without having to worry about importing variables and decyphering their naming scheme.